src/couch/src/couch_bt_engine.erl

```diff
@@ -1235,3 +1236,10 @@ finish_compaction_int(#st{} = OldSt, #st{} = NewSt1) ->
     {ok, NewSt2#st{
         filepath = FilePath
     }, undefined}.
+
+
+is_file(Path) ->
+    case file:read_file_info(Path, [raw]) of
+        {ok, #file_info{type = regular}} -> true;
```

Comment:
We might want to handle directories here, and possibly symlinks, just to match on the behavior of `filelib:is_file/1`

Reply:
Good catch.

Reply:
I looked it up and its just regular and directory:

https://github.com/erlang/otp/blob/72f704c7b83643bec889eabe3f9fe378639bb06e/lib/stdlib/src/filelib.erl#L138-L146

So I've added directory but not symlinks.